bindings/wp8/MegaSDK.cpp

```diff
@@ -1843,6 +1843,16 @@ void MegaSDK::getExtendedAccountDetails(bool sessions, bool purchases, bool tran
     megaApi->getExtendedAccountDetails(sessions, purchases, transactions);
 }
 
+void MegaSDK::queryBandwidthQuota(int64 size, MRequestListenerInterface^ listener)
+{
+    megaApi->queryBandwidthQuota(size, createDelegateMRequestListener(listener));
```

Comment:
This MegaApi function has been refactored to `queryTransferQuota()`. You need to change this call in the binding.